package.json

```diff
@@ -122,11 +122,12 @@
         "ts-jest": "27.1.5",
         "ts-node": "10.0.0",
         "typescript": "4.6.2",
-        "webpack": "5.64.0",
-        "webpack-cli": "4.9.1"
+        "webpack": "^5.94.0",
```

Comment:
Please, do not add webpack as a dependency directly the dependencies for webpack should come from the config package.

Reply:
Removed